src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/allelespecific/AS_ReadPosRankSumTest.java

```diff
@@ -37,9 +38,10 @@
  * </ul>
  *
  */
-//FIXME - this whole class is essentially code duplicated from ReadPosRankSumTest
 public class AS_ReadPosRankSumTest extends AS_RankSumTest implements AS_StandardAnnotation {
 
+    final ReadPosRankSumTest readPosRankSumTest = new ReadPosRankSumTest();
```

Comment:
It looks to me like `getElementForRead()`, at least, could be made static. I suspect that `isUsableRead()` could be refactored to be static as well. Then you wouldn't have to instantiate `ReadPosRankSumTest` like this here.